toolkit/scripts/toolchain/check_manifests.sh

```diff
@@ -36,8 +36,16 @@ write_rpms_from_toolchain () {
     specs=$(cat $TOOLCHAIN_SPEC_LIST_FILE)
     for specName in $specs
     do
-        specFile=$(find $SPECS_DIR/**/$specName.spec)
-        write_rpms_from_spec $specFile $1
+        if [[ $specName == *"msopenjdk"* ]]; then
+            # special case to add msopenjdk-11 which is downloaded and does not have a SPEC
+            jdkfilename=$(cat "$MANIFESTS_DIR/toolchain_$ARCH.txt" | grep $specName)
+            echo "adding special case for: $jdkfilename"
+            echo "$jdkfilename" >> $1
+        else
+            # normal case - search for the RPM filename using the SPEC name
+            specFile=$(find $SPECS_DIR/**/$specName.spec)
```

Comment:
nit: isn't it always `$SPECS_DIR/$specName/$specName.spec`?

Reply:
not always. for example we used to have openjdk8/openjdk8.spec and openjdk8/openjdk8_aarch64.spec